### ISSUE TITLE
New feature for database adaptation

### ISSUE THREAD

**Issue by mimbres** [mimbres/neural-audio-fp#13]
After thinking a bit more, it would be nice to add a few lines to allow loading from other devices. Someone like @Mihonarium might want to load a pre-trained checkpoint and further train with different music data. As mentioned in the paper section 4.2, I've seen small performance gains by using 10% of test-dummy-set as training data. I call this use case as "seen database" to distinguish with general cases of fully "unseen" test set.

...............................................................................................................................................................................................................................................
What's even more strange, the issue with a lot of warnings appears only with `run.py train` and doesn't appear for `generate`.

The notebook: https://gist.github.com/Mihonarium/e3fd355cb560b82373fd2186139f1bc2 (the last cells show that `generate` and training from scratch work).

_Originally posted by @Mihonarium in https://github.com/mimbres/neural-audio-fp/issues/10#issuecomment-878456349_
